tool/jt.rb

```diff
@@ -29,6 +29,9 @@
 GRAAL_DIR = File.expand_path('../graal', TRUFFLERUBY_DIR)
 PROFILES_DIR = "#{TRUFFLERUBY_DIR}/profiles"
 
+CACHE_EXTRA_DIR = File.expand_path('~/.mx/cache/extra')
+FileUtils.mkdir_p(CACHE_EXTRA_DIR) unless Dir.exist?(CACHE_EXTRA_DIR)
```

Comment:
Minor: the condition is not needed since `mkdir_p` will noop if it already exists.

Reply:
Done.